### PR TITLE
Added Styles and background gradient for main page 

### PR DESCRIPTION
**Pull request by inolasv**
## Description

Worked with @nzhan2 @inolasv on #238 

Addresses #<number>

## Screenshots

Screen Shot 2023-10-20 at 4 11 57 PM

### CODE REVIEW

**Review by J164 — APPROVED** [review]
Looks good! I really like the gradient and new font.

File: assets/css/home.css
Comment (neha-vard) [inline assets/css/home.css:1]: We probably don't want to commit this file (this is from the master branch)

File: assets/css/style.css
Comment (neha-vard) [inline assets/css/style.css:1]: Same as above.

File: assets/design-vectors/sparkle.svg
Comment (neha-vard) [inline assets/design-vectors/sparkle.svg:1]: Thank you for adding these! Could we also add an .svg for the keyboard key (the one that has the social media)?

File: index.html
Comment (neha-vard) [inline index.html:1]: Same thing, we don't want to commit this file.

File: src/styles/globals.css [hunk 1]
```diff
@@ -89,12 +91,15 @@ body {
...
       to bottom,
       transparent,
       rgb(var(--background-end-rgb))
     )
-    rgb(var(--background-start-rgb));
+    rgb(var(--background-start-rgb)); */
+    background: linear-gradient(#FFB2DA, #CBEDFF);
+    background-attachment: fixed;
+    background-repeat: no-repeat;
 }
```
Comment (neha-vard) [inline src/styles/globals.css:103]: This looks great :)

File: src/styles/globals.css [hunk 2]
```diff
@@ -107,3 +112,26 @@ a {
...
+h3 {
+  font-family: "Lato" !important;
+  font-weight: 300;
+  color: black;
+}
+
+
+p {
+  font-family: "Lato", sans-serif;
+}
```
Comment (neha-vard) [inline src/styles/globals.css:137]: Great job styling these in global.css! In the screenshot below, I have an example of the h1, h2, h3, and p as styled currently. I think we can increase the font size on h1 and p, and the font weight on p--could we match these to the Figma?
image

**Review by neha-vard — CHANGES_REQUESTED** [review]
Thank you for working on this! Really love the fonts and gradient you all chose. Just left a few small comments, but this looks great!

**Review by neha-vard — APPROVED** [review]

**Pull request merged**